qmetaobject/Cargo.toml

```diff
@@ -11,13 +11,15 @@ keywords = ["Qt", "QML", "QMetaObject",]
 repository = "https://github.com/woboq/qmetaobject-rs"
 
 [features]
+default = ["log"]
```

Comment:
Thanks.  I'm just concerned about having too many dependencies.  That's also why i wouldn't put it as the default either.

Reply:
I think debugging is essential, especially on such multi-lingua projects, and should be on by default. By having it on by default AND providing instructions for activation on the front-page example we are lowering the barrier for newcomers.